packages/edit-site/src/components/sidebar-navigation-screen-pattern/use-navigation-menu-content.js

```diff
@@ -47,6 +52,11 @@ function getBlocksOfTypeFromBlocks( targetBlockType, blocks ) {
 export default function useNavigationMenuContent( postType, postId ) {
 	const { record } = useEditedEntityRecord( postType, postId );
 
+	const blocks =
+		record && record.content && typeof record.content !== 'function'
+			? parse( record.content )
+			: [];
+
```

Comment:
Moving this code block after the early return check below might be a good idea. Parsing blocks can be expensive; let's avoid it when unnecessary.